### PR TITLE
feat: add documentation comments for methods used to mount a UFS path to an Alluxio path

### PR DESCRIPTION
**Pull request by abnerferreira0212**
# Pull Request Merge Checklist
           
Task backlog identifier related to this PR: ---

---

## As a DEVELOPER, I confirm that this PR **doesn't contain**:
- [x] Code lines with syntax errors
- [x] Methods and variables names that aren't clear or doesn't mean anything
- [x] Code that is out of the standard (Example: **PEP-8** for Python code)
- [x] Files with **imports** errors or without any utility
- [x] **TODOs** left behind
- [x] Tests that are being skipped
- [x] Methods placed on incorrect files or folders (Example: methods that are local, but should be placed on /utils)
- [x] Functions, methods or API responses that are publicly returning sensitive information

## As a DEVELOPER, I confirm that I:
- [x] Followed the Simbiose's standard for branch and commit naming
- [x] Concluded the task and achieved the goals according to the task planning I defined before starting to code
- [x] Added all necessary database changes to the [SQL Migrations](https://github.com/s1mbi0se/sql-migrations)
- [x] Created tests for every new flow and method on this Pull Request
- [x] Added the necessary DEBUG logs
- [x] Checked my code using the Simbiose's Corner Cases Guide
- [x] Merged the **develop** branch to this PR/branch before sending the PR for review
- [x] Solved all possible merge conflicts before sending the PR for review
- [x] Corrected all the problems and code smells identified by the quality tools
- [x] Manually executed the necessary tests locally to make sure my changes didn't break anything
- [x] All unitary and integration tests ran by the CI system are passing
- [x] Check the performance of my changes or additions and they didn't make the system slower (I profiled the code)
- [x] Updated the necessary documentation (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [x] Made the self-review of the code before sending the PR for review
- [x] Sent this PR for a peer-review
- [x] Made all the corrections appointed during the peer-review
- [x] Confirmed that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [x] Checked all GitHub Actions related to this PR that can block my merge

---  

## As the TEAM MANAGER, I confirm that I:
- [x] Checked that the code was developed following the best practices for the language, with the best possible readability and using the less possible quantity of code lines and complexity. 
- [x] Checked if the Pull Request and commit messages follow the Simbiose’s defined standard 
- [x] Checked if there is no possible merge conflicts before reviewing the PR
- [x] Checked if the purpose of the task was reached with this PR
- [x] Checked that the code was developed following the best practices for the language, with the best possible readability and using the less possible quantity of code lines and complexity. 
- [x] Checked if all related PRs were created when other components were affected by this task (e.g. database changes on [SQL Migrations](https://github.com/s1mbi0se/sql-migrations))
- [x] Checked that were created unit, integration and chaos tests for every new flow and method on this Pull Request
- [x] Checked that no sensible information (e.g. users security data, internal security data, etc) was exposed by this PR
- [x] Checked that code coverage for this PR is higher than 80%.
- [x] Checked that there are no problems and code smells identified by the quality and security tools
- [x] Checked that all files were created on the proper folders according to the Simbiose’s defined standard
- [x] Checked that all classes, methods and variables on this PR follow the Simbiose’s pattern for this programming language
- [x] Checked that all unitary and integration tests ran by the CI system are passing
- [x] Checked that there are no performance issues created by these changes or additions
- [x] Checked that all documentation is up to date according to this change (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [x] Checked that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [x] Checked all GitHub Actions related to this PR that can block the merge

---  

## As the QUALITY RESPONSIBLE, I confirm that I:
- [x] Checked if the Pull Request and commit messages follow the Simbiose’s defined standard 
- [x] Checked if there is no possible merge conflicts before reviewing the PR
- [x] Checked that were created unit, integration and chaos tests for every new flow and method on this Pull Request
- [x] Checked that no sensible information (e.g. users security data, internal security data, etc) was exposed by this PR
- [x] Checked that code coverage for this PR is higher than 80%.
- [x] Checked that there are no problems and code smells identified by the quality and security tools
- [x] Checked that all files were created on the proper folders according to the Simbiose’s defined standard
- [x] Checked that all classes, methods and variables on this PR follow the Simbiose’s code standard for this programming language
- [x] Checked that all unitary and integration tests ran by the CI system are passing
- [x] Checked that there are no performance issues created by these changes or additions
- [x] Checked that all documentation is up to date according to this change (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [x] Checked that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [x] Checked all GitHub Actions related to this PR that can block the merge

## Important links:
1. [Simbiose Quality Guidelines](https://github.com/s1mbi0se/company/wiki/Quality-Topics#table-of-contents)
2. [Metadata Database Modeling](https://docs.google.com/spreadsheets/d/1Y8_k5xV86kTjgfjbi-PGsJcubUKhWlT2x8ecpnMlyR0/edit?usp=sharing)
3. [Simbiose's Corner Cases Guide](https://www.simbioseventures.com/documentos-e-guias/procedimentos-e-checklists/corner-cases-guide)

### CODE REVIEW

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 1]
```diff
@@ -1184,6 +1184,20 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.SERVER)
           .build();
+  public static final PropertyKey B2_ACCESS_KEY =
```
Comment (caiopavanelli) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java:1187]: remove this commit

File: core/common/src/main/java/alluxio/util/ConfigurationUtils.java [hunk 2]
```diff
@@ -44,6 +44,7 @@
 import com.google.common.base.Splitter;
 import com.google.common.collect.Lists;
 import com.google.common.collect.Sets;
+import io.grpc.Channel;
```
Comment (caiopavanelli) [inline core/common/src/main/java/alluxio/util/ConfigurationUtils.java:47]: why this dependency was added?

File: pom.xml [hunk 3]
```diff
@@ -161,6 +161,7 @@
     <update.check.enabled>true</update.check.enabled>
     <update.check.host>https://diagnostics.alluxio.io</update.check.host>
     <findbugs.skip>false</findbugs.skip>
+    <b2.version>3.1.0</b2.version>
```
Comment (caiopavanelli) [inline pom.xml:164]: remove this commit

File: underfs/b2/pom.xml [hunk 4]
```diff
@@ -0,0 +1,87 @@
+<!--
```
Comment (caiopavanelli) [inline underfs/b2/pom.xml:1]: remove this commit

File: underfs/b2/src/main/java/alluxio/underfs/b2/B2InputStream.java [hunk 5]
```diff
@@ -0,0 +1,147 @@
+/*
```
Comment (caiopavanelli) [inline underfs/b2/src/main/java/alluxio/underfs/b2/B2InputStream.java:1]: remove this commit

File: underfs/b2/src/main/java/alluxio/underfs/b2/B2UnderFileSystem.java [hunk 6]
```diff
@@ -0,0 +1,286 @@
+/*
```
Comment (caiopavanelli) [inline underfs/b2/src/main/java/alluxio/underfs/b2/B2UnderFileSystem.java:1]: remove this

File: underfs/b2/src/main/java/alluxio/underfs/b2/B2UnderFileSystemFactory.java [hunk 7]
```diff
@@ -0,0 +1,61 @@
+/*
```
Comment (caiopavanelli) [inline underfs/b2/src/main/java/alluxio/underfs/b2/B2UnderFileSystemFactory.java:1]: remove this

File: underfs/b2/src/main/java/alluxio/underfs/b2/B2Utils.java [hunk 8]
```diff
@@ -0,0 +1,49 @@
+/*
```
Comment (caiopavanelli) [inline underfs/b2/src/main/java/alluxio/underfs/b2/B2Utils.java:1]: remove this

File: underfs/b2/src/main/resources/META-INF/services/alluxio.underfs.UnderFileSystemFactory [hunk 9]
```diff
@@ -0,0 +1,12 @@
+#
```
Comment (caiopavanelli) [inline underfs/b2/src/main/resources/META-INF/services/alluxio.underfs.UnderFileSystemFactory:1]: remove this

File: underfs/b2/src/test/java/alluxio/underfs/b2/B2InputStreamTest.java [hunk 10]
```diff
@@ -0,0 +1,103 @@
+/*
```
Comment (caiopavanelli) [inline underfs/b2/src/test/java/alluxio/underfs/b2/B2InputStreamTest.java:1]: remove this

File: underfs/b2/src/test/java/alluxio/underfs/b2/B2UnderFileSystemFactoryTest.java [hunk 11]
```diff
@@ -0,0 +1,37 @@
+/*
```
Comment (caiopavanelli) [inline underfs/b2/src/test/java/alluxio/underfs/b2/B2UnderFileSystemFactoryTest.java:1]: remove this

File: underfs/b2/src/test/java/alluxio/underfs/b2/B2UtilsTest.java [hunk 12]
```diff
@@ -0,0 +1,56 @@
+/*
```
Comment (caiopavanelli) [inline underfs/b2/src/test/java/alluxio/underfs/b2/B2UtilsTest.java:1]: remove this

File: underfs/pom.xml [hunk 13]
```diff
@@ -34,6 +34,7 @@
     <module>wasb</module>
     <module>web</module>
     <module>ozone</module>
+    <module>b2</module>
```
Comment (caiopavanelli) [inline underfs/pom.xml:37]: remove this

File: core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java [hunk 14]
```diff
@@ -2690,6 +2690,46 @@ public void updateMount(AlluxioURI alluxioPath, MountContext context)
...
+   * where files and folders created in Alluxio space
+   * under the path will be backed by a corresponding
+   * file or folder in the under storage path.
+   * <p>
+   * Increments {@link DefaultFileSystemMaster.Metrics#MOUNT_OPS},
+   * which keeps track of the number of mount operations made. Uses
+   * {@link LockingScheme} to determine that the path should be locked
+   * with the lock pattern {@link LockPattern#WRITE_EDGE}.
+   * <p>
+   * Attempts to instantiate {@link RpcContext} through {@link DefaultFileSystemMaster#createRpcContext()}
```
Comment (gpang) [inline core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java:2706]: I think this is too detailed on the implementation of the method. Comments about implementation should go within the code itself.

The method documentation should include the behavior or contract of the method, but not necessarily how it achieves that behavior.

File: core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java [hunk 15]
```diff
@@ -2690,6 +2690,46 @@ public void updateMount(AlluxioURI alluxioPath, MountContext context)
...
+   *                    1) rpcContext, the RpcContext;
+   *                    2) inodePath, the LockedInodePath;
+   *                    3) ufsPath, the provided UFS path to mount to Alluxio;
+   *                    4) context, the provided object of type {@link MountContext},
+   *                    used to merge and wrap {@link MountPOptions}.
+   *
+   * @param alluxioPath the Alluxio path to mount to
+   * @param ufsPath     the UFS path to mount
+   * @param context     the mount context
+   * @throws FileAlreadyExistsException If an Alluxio path already exists.
```
Comment (gpang) [inline core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java:2726]: Typically, the convention in Alluxio is to not javadoc these exceptions, since the exceptions are usually self-explanatory.

File: core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java [hunk 16]
```diff
@@ -2690,6 +2690,46 @@ public void updateMount(AlluxioURI alluxioPath, MountContext context)
     }
   }
 
+  /**
+   * Mounts an under storage path to an Alluxio path.
```
Comment (gpang) [inline core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java:2694]: The convention within Alluxio codebase is to not add javadocs to methods which are `@Override` methods.
